bokehjs/src/coffee/document.coffee

```diff
@@ -573,7 +573,7 @@ class Document
       throw new Error("JSON object has wrong type #{typeof json}")
     py_version = json['version']
     versions_string = "Library versions: JS (#{js_version})  /  Python (#{py_version})"
-    if js_version.split('-')[0] != py_version.split('-')[0]
+    if js_version != py_version
```

Comment:
None of the release tags I've seen use a dash in the name. Regardless, if we are calling `version_update.py` it will automatically change the text values of `version.coffee` and `package.json`, and so all that is needed is a simple equality check.